src/index.ts

```diff
@@ -90,6 +101,23 @@ export class Sql {
     return value;
   }
 
+  bind(...params: Value[]) {
```

Comment:
Doesn't technically need this method if you know what you're doing, but the array length safety and merging with any non-params seems like a good pattern to enforce.